src/PhpSpreadsheet/Collection/Cells.php

```diff
@@ -301,6 +301,14 @@ private function getUniqueID()
         return uniqid('phpspreadsheet.', true) . '.';
     }
 
+    /**
+     * @param mixed $str
+     */
+    private static function forceString($str): string
+    {
+        return (string) $str;
+    }
+
```

Comment:
In this case, I think we can agree that Scrutinizer is wrong, and we don't even need the casting. (And I still can find only one usage :stuck_out_tongue_closed_eyes: )

Reply:
Okay, so I can't count :-(
I don't think it's super important for this change to be part of the new release. I would like to clean it up afterwards rather than make changes now in a rush. One question - having agreed that Scrutinizer is wrong, how would we go about merging it even if Scrutinizer fails it? I thought the relevant button was grayed out until all tests passed.

Reply:
> the relevant button was grayed out

Correct... until now. Administrators could always bypass this check, and I just changed the settings so that specifically Scrutinizer can be bypassed too, by everyone.

But it is still a check that we **should take into account manually**, because it is still useful and does things that PHPStan doesn't (unused code, stats over time, complexity analysis, etc.). But as you experienced it also has some false positive, that sometimes makes it more annoying than helpful.

Let me know if that doesn't solve the situation in the coming days.

Reply:
ForceString is now deleted. PhpStan had also objected to the original because str_replace might return an array, but assigning the result of str_replace to a docblock typehinted variable was sufficient for PhpStan; maybe it will satisfy Scrutinizer too.